api.py

```diff
@@ -14,6 +14,15 @@
 
 app = FastAPI()
 
+ALLOWED_FULL_DOMAINS = {
```

Comment:
If we're adding more constants here, it might be worth logically grouping these together with the existing consts, and placing them all near the top of the module?

Reply:
Good Point. I have pushed some changes, reformatting the constants.